src/css/base.styl

```diff
@@ -506,14 +506,15 @@ nav ul
 .win.centered
   position: fixed
   top: 30%
-  left: 45%
+  left: 35%
+  width: 40%
+  text-align: center
```

Comment:
Can we get some rounded corners on the win box as well (as most other panels have) :)

Reply:
done - just not in this bit of code